packages/slice-machine/bin/start.js

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env node
 
-require("../build/scripts/start/index.js");
+// eslint-disable-next-line @typescript-eslint/no-var-requires, @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
```

Comment:
:o why did you had issue with the linter on that, i'm curious. it's like one of those files we never touch.

Reply:
I changed the start script to be a function :)

Reply:
In which purpose :) ?

Reply:
`../build/scripts/start/index.js`

now exports a function, so I could test or run it :)